src/pages/charter.js

```diff
@@ -0,0 +1,309 @@
+import Layout from "../layouts";
+import React from 'react'
+import { Helmet } from "react-helmet";
+
+const pageMetaTitle = 'The eBPF Foundation Charter'
+const pageMetaDescription = 'eBPF is a revolutionary technology that can run sandboxed programs in the Linux kernel without changing kernel source code or loading a kernel module.'
+
+const CharterPage = ({ location: {pathname} }) => {
+  return <Layout path={pathname}>
+    <div className="page-wrapper page-charter">
+    <Helmet
+        title={pageMetaTitle}
+
+        meta={[
+          {name: "keywords", content: "ebpf, bpf, contribute, getting started, community"},
+          {name: "type", property: "og:type", content: "website"},
+          {name: "url", property: "og:url", content: "https://ebpf.io/contribute/"},
```

Comment:
This field doesn't look right, please double-check. Can you also make sure that all relevant metadata fields are updated for the new page?